rockset_test.go

```diff
@@ -119,6 +119,16 @@ func TestRockClient_withAPIServerEnv(t *testing.T) {
 	log.Debug().Str("org", org.GetDisplayName())
 }
 
+func TestRockClient_withBearerToken(t *testing.T) {
+	token := "bearer token"
+	org := "rockset org"
+
+	rc, err := rockset.NewClient(rockset.WithBearerToken(token, org), rockset.WithAPIServer("apiserver"))
+	require.NoError(t, err)
+	assert.Equal(t, token, rc.Token)
+	assert.Equal(t, org, rc.Organization)
+}
+
```

Comment:
You most probably want to have a few more tests for the case where you have both the token and API key, and for the case when you have none.